servant-client/test/Servant/ClientSpec.hs

```diff
@@ -30,7 +30,7 @@ module Servant.ClientSpec where
 import           Control.Applicative        ((<$>))
 #endif
 import           Control.Arrow              (left)
-import           Control.Concurrent         (forkIO, killThread, ThreadId)
+import           Control.Concurrent         (ThreadId, forkIO, killThread)
```

Comment:
But it's a pain to be consistent about formatting without using automated tools, and automated tools will touch parts that are inconsistent. My opinion is that we should aim to follow some stylish-haskell configuration, and *then* running the tool wouldn't change anything besides what you touched.  But if we still haven't reached that stage, we'll have issues like this.